consult-dir.el

```diff
@@ -241,6 +295,36 @@ Entries that are also in the list of projects are removed."
                                 (consult-dir--project-dirs)))))
   "Project directory source for `consult-dir--pick'.")
 
+(defvar consult-dir--source-tramp-local
+  `(:name     "Local"
+    :narrow   ?l
+    :category file
+    :face     consult-file
+    :history  file-name-history
+    :enabled  ,(lambda () consult-dir-include-tramp-hosts)
+    :items    ,consult-dir-tramp-local-hosts)
+  "Local host candidate source for `consult-dir'.")
+
+(defvar consult-dir--source-tramp-ssh
+  `(:name     "SSH Config"
+    :narrow   ?s
+    :category file
+    :face     consult-file
+    :history  file-name-history
+    :enabled  ,(lambda () consult-dir-include-tramp-hosts)
+    :items    ,#'consult-dir--tramp-ssh-hosts)
+  "SSH Config candiadate source for `consult-dir'.")
+
+(defvar consult-dir--source-tramp-docker
```

Comment:
Remove this source, add to README & to wiki instead.

Reply:
Should `consult-dir--tramp-docker-hosts` also be removed and moved to documentation then?

Reply:
Yes.

Specifically, I think the wiki would be the best place to put recipes for additional sources that require packages not included in Emacs. We can link to these from the README.

This is similar to the organization of recipes for [additional actions in Embark](https://github.com/oantolin/embark/wiki/Additional-Actions) or [addtional sources in Consult](https://github.com/minad/consult/blob/main/README.org#multiple-sources).

Reply:
OK, I'll remove the README entry and add to the wiki :)